sass/basics.scss

```diff
@@ -37,6 +37,33 @@ a {
   color: red;
 }
 
+.tipMessage {
+  border: 1px solid #c3e0c9;
+  background-color: #f3f9f4;
+  color: #526d8a;
+  font-size: smaller;
+  padding: 0.5em;
+  margin-bottom: 0.5em;
```

Comment:
Nit: You might want to consider factoring out a `.message` class, along these lines:

```
.message {
  padding: 0.5em;

  .tip {
    background-color: #f3f9f4
  }
}
```